File: tasking/sources/tasking.11.c
/*
* @@name:	tasking.11
* @@type:	C
* @@operation:	compile
* @@expect:	success
* @@version:	omp_3.1
*/
#include <stdio.h>
void foo ( )
{
   int x = 2;
   #pragma omp task shared(x) mergeable
   {
      x++;
   }
   #pragma omp taskwait
   printf("%d\n",x);  // prints 3
}

File: tasking/sources/tasking.11.f90
! @@name:	tasking.11
! @@type:	F-free
! @@compilable:	yes
! @@linkable:	no
! @@expect:	success
! @@version:    omp_3.1
subroutine foo()
  integer :: x
  x = 2
!$omp task shared(x) mergeable
  x = x + 1
!$omp end task
!$omp taskwait
  print *, x     ! prints 3
end subroutine
